vignettes/GGIR.Rmd

```diff
@@ -532,6 +524,39 @@ use `GGIR`. So, if you would like to add annotations in the file, e.g.
 in the fourth column, then you will need to store it somewhere outside
 the output folder and explicitly point to it with `configfile` argument.
 
+### Tailoring GGIR to study protocol
+
+Usual study protocols to collect multiday accelerometer raw data last one
```

Comment:
I am not sure I agree with the first sentence. Most studies nowadays come with 24/7 wear protocols and often last longer than a week. This first sentence could give the impression that GGIR assumes that most studies only collect one week of data.
Instead, I think it is better to just state that there is a large variation in study designs and GGIR facilitates accounting for this variation.

Reply:
Done.